jameda_interface_config.js

```diff
@@ -57,10 +57,8 @@ var interfaceConfig = {
         'videoquality',
         'filmstrip',
         'help',
-        'notes'
-
-        // FIXME to be implemented
-        // 'fileTransfer'
+        'notes',
```

Comment:
I'm not sure if help and closedcaptions buttons are needed. 
Don't we add tileview button here?